test/cmake/compress-and-verify.cmake

```diff
@@ -101,24 +113,47 @@ macro(diff src1 src2)
     endif()
 endmacro()
 
+
+macro(exec_streams cmd1 src1 dst1)
+    execute_process(COMMAND ${CMAKE_COMMAND}
+        "-DCOMMAND=${cmd1}"
+        -DINPUT=${src1}
+        -DOUTPUT=${dst1}
+        "-DSUCCESS_EXIT=${SUCCESS_EXIT}"
+        -P ${CMAKE_CURRENT_LIST_DIR}/run-and-redirect.cmake
+        RESULT_VARIABLE CMD_RESULT
+        COMMAND_ECHO STDOUT)
+endmacro()
+
+macro(exec_files cmd1 src1)
```

Comment:
Do we need number 1 after each arg name?

Reply:
I am unaware of the list of reserved keywords in cmake, therefore didn't want to name them `cmd` and `src`.
Changing them to `tcmd` and `tsrc`.